content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+
+When we talk about linear regression, we'll probably want to discuss this equation in a slightly alterred form: $$y = \beta_0 + \beta_1 x_1$$. We'll discuss what these terms mean in a moment but they relate exactly to a y-intercept ($\beta_0$), a slope ($\beta_1$) and an x-value. 
+
+## Ordinary Least Squares (OLS)
+
+We mentioned above we're putting a line through some points as best we can. But how do we define this "best as we can"? Well, the version of "as best we can" that will be addressed here is the method of least squares. If we imagine a scatter of plots like this: 
+
+```{r, echo = FALSE, warning = FALSE}
+x = c(1:100)
+y = rep(NA, 100)
+for(i in 1:100) {
+  y[i] = 1.3 * x[i] + 10 + rnorm(1, mean = 0, sd = 23)
+}
+
+df = data.frame(
+  x = x, y = y
+)
+
+library(tidyverse)
+library(ggthemes)
+ggplot(df) + 
+  geom_point(aes(x = x, y = y), fill = "red", colour = "black", shape = 21, 
+             size = 3) + 
+  ggthemes::theme_base()
+```
+
+It's clear that we can't actually make our line go <em>directly</em> through very many of them, but perhaps we can figure out some way to make the line <em> as close to as many points as possible</em>. We'll do that by imagining we draw a line at random, and calculate the distance between every single point and the line itself. These are called <strong>residuals</strong>. Then we'll square each one, and take the sum of all those squares. 
+
+You might ask, why square it? The short answer to that question is that squaring it makes a large number of mathematical tools available to us. 
+
+It now follows somewhat logically that we want to find a value for the slope of our line, $\beta_1$ that <strong>minimizes</strong> this sum of the squared residuals. What we are doing here is actually a process called <em>estimation</em> wherein we choose some method that allows us to compare candidate values for $\beta_1$ with other values. Ordinary Least Squares is actually just a specific version of Maximum Likelihood Estimation, which we'll discuss below. 
+
+## Linear Model
+
+As in our previous work, we will certainly NOT (!!!!) just jump into writing code, as that's the best way to ensure a poorly done analysis. First, we have our question. Let's frame it as a hypothesis. It seems logical here to state that we think temperature will positively vary with ice cover. So a null hypothesis may be that <em> there is no relationship between air temperature and ice cover </em>. We have decided we want to use linear regression to tackle this challenge, so consider the assumptions of the method: 
```

Comment:
This doesn't make a clear distinction between correlation and regression. The question is formulated as a correlation. There should be some hypothesis of causation